### PR TITLE
Add Emacs

### PR DESCRIPTION
**Pull request by gowikel**
This adds information about the editor Emacs and some configuration using the Spacemacs init framework.

### CODE REVIEW

**Review by Ollymid — COMMENTED** [review]
Couple of typos that I found, the rest need to be added as exclusions so our TypoCi service doesn't pick out the words as typos

To do that you'll have to add a `.typo-ci.yml` file to the root of this repository, this is a config file for TypoCi and you should then add all the words highlighted by TypoCi in this PR as excluded words  (Gedit etc)

There is an example typo-ci config file in [their documentation](https://typoci.com/documentation) or you can use the [POPSSS one](https://github.com/wegotpop/popsss/blob/master/.typo-ci.yml) for guidance :+1:

File: practices/text-editors/emacs.md [hunk 1]
```diff
@@ -0,0 +1,287 @@
...
+or [Purcel's config](https://github.com/purcell/emacs.d)
+are good starting points
+
+
+### Mandatory plugins
+
+POP uses the [Black](https://black.readthedocs.io/en/stable/), [Prettier](https://prettier.io/) and
+[EditorConfig](https://editorconfig.org/), so you should install and integrate them on your emacs configuration.
+
+Therefore, I do recommend the following extensions:
```
Comment (rrees) [inline practices/text-editors/emacs.md:29]:
```suggestion
Therefore install the following extensions:
```

This is the mandatory section so you're not recommending anything

File: practices/text-editors/emacs.md [hunk 2]
```diff
@@ -0,0 +1,287 @@
...
+
+* [python-black](https://github.com/wbolster/emacs-python-black)
+* [PrettierJs for Emacs](https://github.com/prettier/prettier-emacs)
+* [EditorConfig Emacs plugin](https://github.com/editorconfig/editorconfig-emacs)
+
+All of them are accessible at [MELPA](https://melpa.org/#/).
+
+
+### I am new to Emacs. A getting starting guide.
```
Comment (rrees) [inline practices/text-editors/emacs.md:38]: Just link to a tutorial here for the curious. Assume that someone is here on this page to learn what to do to use Emacs at POP.

File: practices/text-editors/emacs.md [hunk 3]
```diff
@@ -0,0 +1,287 @@
+---
+layout: page
+title: Setting Up Visual Studio Code
```
Comment (rrees) [inline practices/text-editors/emacs.md:3]:
```suggestion
title: Setting Up Emacs
```

File: practices/text-editors/index.md [hunk 4]
```diff
@@ -4,5 +4,5 @@ title: Text Editor Setup
 ---
 
 
-
-* [Setting Up VSCode](setting-up-vscode.md)
\ No newline at end of file
+* [Emacs](emacs.md)
```
Comment (rrees) [inline practices/text-editors/index.md:7]:
```suggestion
* [Setting Up Emacs](emacs.md)
```

**Review by rrees — CHANGES_REQUESTED** [review]

**Pull request closed without merging**